CSharpMath.Tests/PreTypesetting/MathListBuilderTest.cs

```diff
@@ -973,32 +972,35 @@ public class MathListBuilderTest {
 
       Assert.Single(list);
       var latex = MathListBuilder.MathListToString(list);
-      Assert.Equal(input, RemoveWhiteSpacesFromString(latex));
+      Assert.Equal(@"\begin{array}{lr}x^2&\: x<0\\ x^3&\: x\geq 0\end{array}", latex);
     }
 
     [Fact]
-    public void TestListToString_int_a_to_b() {
-      TestLatexToListAndBack(@"\int_a^b");
+    public void TestListToString_integral_a_to_b() {
+      var input = @"\int_a^b";
+      var list = MathLists.FromString(input);
+
+      Assert.Single(list);
+      var latex = MathListBuilder.MathListToString(list);
+      Assert.Equal(@"\int _a^b", latex);
     }
 
     [Fact]
-    public void TestListToString_int_wdf() {
-      TestLatexToListAndBack(@"\int_wdf=\int_{\partial w}f");
+    public void TestListToString_integral_wdf() {

```

Comment:
`wdf` still doesn't give the code readers meaning. Most people would assume it's an abbreviation, but it's actually just a random collection of variables.